backend/FwLite/LcmCrdt/CrdtMiniLcmApi.cs

```diff
@@ -22,39 +18,39 @@ public class CrdtMiniLcmApi(DataModel dataModel, JsonSerializerOptions jsonOptio
 
 
     private IQueryable<Entry> Entries => dataModel.GetLatestObjects<Entry>();
-    private IQueryable<CrdtComplexFormComponent> ComplexFormComponents => dataModel.GetLatestObjects<CrdtComplexFormComponent>();
-    private IQueryable<CrdtComplexFormType> ComplexFormTypes => dataModel.GetLatestObjects<CrdtComplexFormType>();
+    private IQueryable<ComplexFormComponent> ComplexFormComponents => dataModel.GetLatestObjects<ComplexFormComponent>();
+    private IQueryable<ComplexFormType> ComplexFormTypes => dataModel.GetLatestObjects<ComplexFormType>();
     private IQueryable<Sense> Senses => dataModel.GetLatestObjects<Sense>();
     private IQueryable<ExampleSentence> ExampleSentences => dataModel.GetLatestObjects<ExampleSentence>();
     private IQueryable<WritingSystem> WritingSystems => dataModel.GetLatestObjects<WritingSystem>();
     private IQueryable<SemanticDomain> SemanticDomains => dataModel.GetLatestObjects<SemanticDomain>();
-    private IQueryable<Objects.PartOfSpeech> PartsOfSpeech => dataModel.GetLatestObjects<Objects.PartOfSpeech>();
+    private IQueryable<PartOfSpeech> PartsOfSpeech => dataModel.GetLatestObjects<PartOfSpeech>();
 
     public async Task<WritingSystems> GetWritingSystems()
     {
         var systems = await WritingSystems.ToArrayAsync();
         return new WritingSystems
         {
             Analysis = systems.Where(ws => ws.Type == WritingSystemType.Analysis)
-                .Select(w => ((MiniLcm.Models.WritingSystem)w)).ToArray(),
+                .Select(w => ((WritingSystem)w)).ToArray(),
             Vernacular = systems.Where(ws => ws.Type == WritingSystemType.Vernacular)
-                .Select(w => ((MiniLcm.Models.WritingSystem)w)).ToArray()
+                .Select(w => ((WritingSystem)w)).ToArray()
         };
     }
 
-    public async Task<MiniLcm.Models.WritingSystem> CreateWritingSystem(WritingSystemType type, MiniLcm.Models.WritingSystem writingSystem)
+    public async Task<WritingSystem> CreateWritingSystem(WritingSystemType type, WritingSystem writingSystem)
     {
         var entityId = Guid.NewGuid();
         var wsCount = await WritingSystems.CountAsync(ws => ws.Type == type);
         await dataModel.AddChange(ClientId, new CreateWritingSystemChange(writingSystem, type, entityId, wsCount));
         return await dataModel.GetLatest<WritingSystem>(entityId) ?? throw new NullReferenceException();
     }
 
-    public async Task<MiniLcm.Models.WritingSystem> UpdateWritingSystem(WritingSystemId id, WritingSystemType type, UpdateObjectInput<MiniLcm.Models.WritingSystem> update)
+    public async Task<WritingSystem> UpdateWritingSystem(WritingSystemId id, WritingSystemType type, UpdateObjectInput<WritingSystem> update)
     {
         var ws = await GetWritingSystem(id, type);
         if (ws is null) throw new NullReferenceException($"unable to find writing system with id {id}");
-        var patchChange = new JsonPatchChange<WritingSystem>(ws.Id, update.Patch, jsonOptions);
```

Comment:
Looks like we can remove jsonOptions from the constructor now (and timeProvider) if you want.